### PR TITLE
[PR] The proj files have been updated to enable SourceLink

### PR DESCRIPTION
**Pull request by JTOne123**
CSProj files have been updated to enable SourceLink in your nuget
---

*[This pull request was created with an automated workflow]*

I noticed that your repository and Nuget package are important for our .NET community, but you still haven't enabled SourceLink.

**We have to take 2 steps:**
1) Please approve this pull request and make .NET a better place for .NET developers and their debugging.
2) **Then just upload the .snupkg file** to https://www.nuget.org/ (now you can find the snupkg file along with the .nuget file)

You can find more information about SourceLine at the following links  
https://github.com/dotnet/sourcelink
https://www.hanselman.com/blog/ExploringNETCoresSourceLinkSteppingIntoTheSourceCodeOfNuGetPackagesYouDontOwn.aspx

If you are interesting about this automated workflow and how it works  
https://github.com/JTOne123/GitHubMassUpdater

*If you notice any flaws, please comment and I will try to make fixes manually*

### CODE REVIEW

File: src/Winton.DomainModelling.DocumentDb/Winton.DomainModelling.DocumentDb.csproj [hunk 1]
```diff
@@ -13,8 +13,16 @@
     <RepositoryUrl>https://github.com/wintoncode/Winton.DomainModelling.DocumentDb</RepositoryUrl>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Title>Winton Domain Modelling DocumentDB</Title>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
```
Comment (Choc13) [inline src/Winton.DomainModelling.DocumentDb/Winton.DomainModelling.DocumentDb.csproj:22]: Can this be moved with the rest of the dependencies already listed below?

**Comment by JTOne123** [comment]
fixed

**Comment by Choc13** [comment]
Thanks for this @JTOne123. I just have one more question. Do I need to make sure that the *.snupkg files are pushed to NuGet when the package is built on appveyor?

**Comment by Choc13** [comment]
Oh and one other thing. Is your bot crawling all of GitHub? We have a few .NET packages in wintoncode and it would be great to have this change made to all of them.

**Comment by JTOne123** [comment]
1 I think you should
2 Not all, only these that have nugets on nuget.org (the job has finished, so if you do not see PR right now from me so - no, just give me URLs of that project and I will run job against these repos)

**Comment by Choc13** [comment]
Interesting, we have a handful of projects that are published to NuGet from GitHub such as:
* [Winton.Extensions.Configuration.Consul](https://github.com/wintoncode/Winton.Extensions.Configuration.Consul)
* [Winton.AspNetCore.Seo](https://github.com/wintoncode/Winton.AspNetCore.Seo)
* [Winton.Extensions.Threading.Actor](https://github.com/wintoncode/Winton.Extensions.Threading.Actor)
* [Winton.DomainModelling.Abstractions](https://github.com/wintoncode/Winton.DomainModelling.Abstractions)
* [Winton.DomainModelling.AspNetCore](https://github.com/wintoncode/Winton.DomainModelling.AspNetCore)

**Comment by JTOne123** [comment]
done for all your repos

**Pull request merged**